### PR TITLE
[MERGE] event_*: clean the ACLs files to be consistent in all sub-modules

### PR DESCRIPTION
**Pull request by std-odoo**
Purpose
=======
In event, many files have different naming format. We want to clean
them, to group the lines by models before changing and adding a new
event group.

Task ID-2204364
COM PR odoo/odoo#65867
Migration odoo/upgrade#2150

### CODE REVIEW

File: addons/website_event_track_exhibitor/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_chat_room_manager,chat.room.manager,website_jitsi.model_chat_room,event.group_event_manager,1,1,1,1
```
Comment (tde-banana-odoo) [inline addons/website_event_track_exhibitor/security/ir.model.access.csv:2]: Is it new ?

**Review by tde-banana-odoo — COMMENTED** [review]
Riviou

File: addons/website_event_track_exhibitor/security/ir.model.access.csv (hunk 1 above)
Comment (std-odoo) [reply addons/website_event_track_exhibitor/security/ir.model.access.csv:2]: @tde-banana-odoo I think we remove it and after my rebase it recreate the file. I remove it and re-push

**Comment by tde-banana-odoo** [comment]
Don't forget to set PR number to each commit + link upgrade commit.

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-merge

**Pull request closed without merging**